packages/zotero.rb

```diff
@@ -0,0 +1,127 @@
+      system "git config submodule.chrome/content/zotero/xpcom/utilities.url '#{$srcdir}/utilities'"
+      system "git config submodule.chrome/content/zotero/xpcom/translate.url '#{$srcdir}/translate'"
+      system "git config submodule.chrome/content/zotero/locale/csl.url '#{$srcdir}/locales'"
+      system "git -c protocol.file.allow=always submodule update"
+      system "echo 'Step 2: PASSED'"
+
+      Dir.chdir("pdf-reader") {
+        system "git submodule init"
+        system "git config submodule.pdf.js.url '#{$srcdir}/pdf.js'"
+        system "git -c protocol.file.allow=always submodule update"
+      }
+
+      Dir.chdir("pdf-worker") {
+        system "git submodule init"
+        system "git config submodule.pdf.js.url '#{$srcdir}/pdf.js'"
+      }
+
+      system "echo 'Installing LTS node.js...'"
+
+      system "nodebrew install v18.16.1 || true" # if node version is already present or locale is not correct somehow it'd throw an error and exit the script
+      system "nodebrew exec v18.16.1 npm i --legacy-peer-deps"
+    }
+
+    system "ls -l -a"
+
+    Dir.chdir("zotero-standalone-build") {
+      system "git submodule init"
+      system "git config submodule.modules/zotero-libreoffice-integration.url '#{$srcdir}/zotero-libreoffice-integration'"
+      system "git -c protocol.file.allow=always submodule update"
+      system "echo Fetching for standalone building..."
+      system "./fetch_xulrunner.sh -p l"
+      system "./fetch_pdftools"
+    }
+
+    system "mv zotero zotero-client"
+
+    Dir.chdir("zotero-client") {
+      system "Actually building..."
+      system "NODE_OPTIONS=--openssl-legacy-provider nodebrew exec v18.16.1 npm run build"
+    }
+
+    Dir.chdir("zotero-standalone-build") {
+      system "scripts/dir_build -p l"
+    }
+  end
+
+  def self.install                 # the steps required to install the package
+    system "install -dDm755 #{CREW_DEST_PREFIX}/{bin,lib/zotero}"
+    system "mv '#{$srcdir}/zotero-standalone-build/staging/Zotero_linux-x86_64'/* '#{CREW_DEST_PREFIX}/lib/zotero'"
+    system "ln -s '#{CREW_DEST_PREFIX}/lib/zotero/zotero' '#{CREW_DEST_PREFIX}/bin/zotero'"
+    system "install -Dm644 '#{$srcdir}/zotero.desktop' '#{CREW_DEST_PREFIX}/share/applications/zotero.desktop'"
+    system "install -Dm644 '#{CREW_DEST_PREFIX}/lib/zotero/chrome/icons/default/default16.png' '#{CREW_DEST_PREFIX}/share/icons/hicolor/16x16/apps/zotero.png'"
+    system "install -Dm644 '#{CREW_DEST_PREFIX}/lib/zotero/chrome/icons/default/default32.png' '#{CREW_DEST_PREFIX}/share/icons/hicolor/32x32/apps/zotero.png'"
+    system "install -Dm644 '#{CREW_DEST_PREFIX}/lib/zotero/chrome/icons/default/default48.png' '#{CREW_DEST_PREFIX}/share/icons/hicolor/48x48/apps/zotero.png'"
+    system "install -Dm644 '#{CREW_DEST_PREFIX}/lib/zotero/chrome/icons/default/default256.png' '#{CREW_DEST_PREFIX}/share/icons/hicolor/256x256/apps/zotero.png'"
```

Comment:
We can use `FileUtils.mv|ln|install ...` instead of `system "...`.  Examine some other packages to get an idea of the syntax.